bigbluebutton-html5/imports/ui/components/connection-status/service.js

```diff
@@ -202,15 +205,17 @@ const getConnectionStatus = () => {
     };
   });
 
-  return Users.find(
-    { meetingId: Auth.meetingID },
-    { fields:
+  return UsersPersistentData.find(
+    {},
```

Comment:
Shouldn't we fetch from the current `meetingId`?

Reply:
@Tainan404 did not include `meetingId` in the published data about UsersPersistentData.
I think it would be simpler if we published `meetingId` field there and kept it unchanged here

Reply:
https://github.com/bigbluebutton/bigbluebutton/blob/develop/bigbluebutton-html5/imports/api/users-persistent-data/server/publishers.js#L21

Reply:
Hmmm, ok. I remember @tdjac0bs mentioning `meetingId` and I did find a good key to have along. Another things that seems to not be handled by the UsersPersistentData collection is the role change.

Reply:
(but this last one is something we can discuss later on)

Reply:
Yes, I tried to fetch from current meeting but got empty results, then I found that we don't really have the `meetingId` in `UsersPersistentData`

Reply:
added `meetingId` to persistent data

Reply:
I didn't add the `meetingId` because in publisher we filter the user by the `meetingId`, so, we don't need filter it in the client, also the `meetingId` is stored in the `Auth` singleton if we need use it in client.